frontend/packages/console-shared/src/utils/keyword-filter.ts

```diff
@@ -0,0 +1,23 @@
+import * as _ from 'lodash';
+
+export const keywordFilter = <T>(
+  filterText: string,
+  items: T[],
+  compareFunction: (keyword: string, item: T) => boolean,
+): T[] => {
+  if (!filterText) {
+    return items;
+  }
+  const keywords = _.uniqBy(filterText.match(/\S+/g), (w) => w.toLowerCase());
```

Comment:
Is your intent to lowercase the keywords here? I don't believe this will do that, it will just use `toLowerCase` for the uniqueness check.

Reply:
Sorry I assumed it would save the lowercase strings.
Updated to use `map`.